opentelemetry_api/opentelemetry/context/traceparent.h

```diff
@@ -0,0 +1,51 @@
+/* Copyright 2019, OpenTelemetry Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#ifndef OPENTELEMETRY_CONTEXT_TRACEPARENT_H_
```

Comment:
(this came up in the meeting) At least Ryan and I are fans of `#pragma once`

Do we need to support older compilers where we need to use the header guard? If so, which ones?

Reply:
I've never seen a compiler that doesn't support it. Looking at the portability section of https://en.wikipedia.org/wiki/Pragma_once even old Sun and IBM compilers work with it.

I'd say adopt it -- you can always change to include guards if someone complains.

Reply:
I vote for the pragma then. :)